nnvm/tests/python/frontend/darknet/test_forward.py

```diff
@@ -50,14 +50,16 @@ def _download(url, path, overwrite=False, sizecompare=False):
 _download(DARKNETLIB_URL, DARKNET_LIB)
 LIB = __darknetffi__.dlopen('./' + DARKNET_LIB)
 
-def _get_tvm_output(net, data):
+def _get_tvm_output(net, data, build_dtype=None):
```

Comment:
Initialize build_dtype to float32 in defenition itself

Reply:
Done